lib/actions/deploy.coffee

```diff
@@ -31,30 +31,17 @@ parseInput = Promise.method (params, options) ->
 
 	return [appName, options.build, source, image]
 
-# Builds and returns a Docker-like progress bar like this:
-# [==================================>               ] 64%
-renderProgress = (percentage, stepCount = 50) ->
-	_ = require('lodash')
-	percentage = Math.max(0, Math.min(percentage, 100))
-	barCount = stepCount * percentage // 100
-	spaceCount = stepCount - barCount
-	bar = "[#{_.repeat('=', barCount)}>#{_.repeat(' ', spaceCount)}]"
-	return "#{bar} #{percentage.toFixed(1)}%"
-
-showPushProgress = (logStreams) ->
-	logging = require('../utils/logging')
-	logging.logInfo(logStreams, renderProgress(0))
-
-updatePushProgress = (percentage, logStreams) ->
-	logging = require('../utils/logging')
-	ansiEscapes = require('ansi-escapes')
+showPushProgress = ->
+	visuals = require('resin-cli-visuals')
+	progressBar = new visuals.Progress('Deploying')
```

Comment:
If we did want to add prefixes, this function could be used: https://github.com/resin-io-modules/stream-logger/blob/master/src/index.ts#L53, and maybe could switch to `[Success]` when completed.